test/regression/RegressionTest.cc

```diff
@@ -53,23 +50,29 @@ void RegressionTest::run(const char* source, const char* triple) {
       simeng::span<char>(reinterpret_cast<char*>(code_), codeSize_), config);
   ASSERT_TRUE(process_->isValid());
   uint64_t entryPoint = process_->getEntryPoint();
-
-  // Allocate memory for the process and copy the full process image to it
-  simeng::span<char> processImage = process_->getProcessImage();
-  processMemorySize_ = processImage.size();
-  if (processMemory_) delete[] processMemory_;
-  processMemory_ = new char[processMemorySize_];
-  std::copy(processImage.begin(), processImage.end(), processMemory_);
-
-  // Create memory interfaces for instruction and data access
-  simeng::FlatMemoryInterface instructionMemory(processMemory_,
+  processMemorySize_ = process_->getProcessImageSize();
```

Comment:
May be useful to keep old comment above this line `// Allocate memory for the process and copy the full process image to it`

Reply:
But because of the optimisation, no extra memory allocation takes place. Only once while creating the LinuxProcess object?

Reply:
PTAL.